string postition_1
string postition_2
string postition_3
